frame/uniques/src/functions.rs

```diff
@@ -256,10 +257,21 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			ensure!(only_buyer == buyer, Error::<T, I>::NoPermission);
 		}
 
+		let mut transfer_amount = price_info.0.clone();
+		if let Ok(royalties) = CollectionRoyaltiesOf::<T, I>::try_get(&collection) {
```

Comment:
using `try_get` is weird to me. normally it would just be `if let Some(r) = CR::<T, I>::get(&c)`. I don't think there is a difference, just saying it is weird.

Reply:
Weirdness removed :)